docs/overview/trustbloc-did-method.md

```diff
@@ -42,11 +45,33 @@ This file contains the current consortium configuration, according to this parti
 ##### Consortium Config Files
 A consortium config file is a JWS, signed by the stakeholders, with the payload being a JSON object containing:
   - The domain name of the consortium
-  - Consortium policy configuration settings
+  - [Consortium policy](#consortium-policy-configuration) configuration settings
   - A list of stakeholders - containing, for each stakeholder:
     - The web domain where its configuration can be found
     - The did:trustbloc DID of the stakeholder, with the associated DID doc in Sidetree on the consortium ledger
+    - The verification key which can be used to verify this stakeholder's signature. The key should match the verification key in the stakeholder's DID doc.
   - The hash of the previous version of this config file
+  
+Example of the format of the configuration data wrapped within the JWS:
+```json
+{
+    "domain": "[consortium domain]",
+    "policy": {},
+    "stakeholders": [
+        {
+            "domain": "stakeholder.one",
+            "did": "[stakeholder one's DID]",
+            "key": "[stakeholder one's verification key]"
```

Comment:
what's the format of this key?

Reply:
It needs to be something that's still accessible if the stakeholder's DID doc is revoked, or if the verification key is replaced in the DID doc. I'm thinking did:key.

Reply:
I've gone with did:key.